src/java/org/apache/cassandra/gms/Gossiper.java

```diff
@@ -65,6 +65,12 @@
 import org.apache.cassandra.utils.JVMStabilityInspector;
 
 import static org.apache.cassandra.config.CassandraRelevantProperties.GOSSIPER_QUARANTINE_DELAY;
+import static org.apache.cassandra.gms.ApplicationState.INTERNAL_ADDRESS_AND_PORT;
+import static org.apache.cassandra.gms.ApplicationState.INTERNAL_IP;
+import static org.apache.cassandra.gms.ApplicationState.NATIVE_ADDRESS_AND_PORT;
+import static org.apache.cassandra.gms.ApplicationState.RPC_ADDRESS;
+import static org.apache.cassandra.gms.ApplicationState.STATUS;
+import static org.apache.cassandra.gms.ApplicationState.STATUS_WITH_PORT;
```

Comment:
can we not do this please. it makes the patch bigger than it needs to be, changing the style of the class unnecessarily. (i'd rather the few long lines)